tests/rolling-upgrade-gce.yaml

```diff
@@ -6,14 +6,16 @@ n_monitor_nodes: 1
 gce_datacenter: 'us-east1-b'
 
 # workloads
+write_stress_during_entire_test: cassandra-stress write no-warmup cl=QUORUM duration=80m -schema 'keyspace=keyspace_entire_test replication(factor=3) compression=LZ4Compressor' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=1000 -pop seq=1..10000000000000 -log interval=5
```

Comment:
We said we don't want to use duration, but to use n=X.
While X is big enough to cover the entire test duration.
This is the only way we can guarntee we dont' override the same keys during the test.

Even if the write will run little bit more than the upgrade process or even less, that's still better than overriding.

Reply:
Also, we don't want to use too big number like the one you put in "seq" because we want to read all keysa afterwards and make sure all there.